rest-tests/src/test/resources/features/integration-sf-db.feature

```diff
@@ -1,21 +1,24 @@
 Feature: sf scenarios
 
+  Background: Clean application state
+    Given clean SF, removes all leads with email: "jdoe@acme.com"
+	And remove all records from DB
+
   @integrations-sf-db
   Scenario: SF action on create - DB integration
-    Given clean before SF to DB, removes user with first name: "John" and last name: "Doe"
-    And create SF connection
+    Given create SF connection
```

Comment:
I'd put also create connection to `Background` section and implement it in create-once fashion.